README.md

```diff
@@ -176,6 +176,70 @@ Releases and publishes the package. Runs tests, lints and builds the project bef
 
 This command uses [`standard-version`](https://github.com/conventional-changelog/standard-version) underneath. The version is automatically inferred from the [conventional commits](https://conventionalcommits.org/).
 
+## Using a linked version of [@discussify-styleguide](https://github.com/ipfs-shipyard/discussify-styleguide)
+
+In some cases, you may want to make changes to Discussify's styleguide at the same time as you work on your project which uses the styleguide. In order to use a local version of [@discussify-styleguide](https://github.com/ipfs-shipyard/discussify-styleguide) and have any styleguide modifications be reflected live on your project, some pages have to be made in our main project.
+
+### Install `postcss-import-webpack-resolver`
+
+Run the following command in your main project
+
+```sh
+$ npm i postcss-import-webpack-resolver
+```
+
+### Update `postcss-preset-moxy` to the latest version
+
+In `package.json` replace the version of the `postcss-preset-moxy` dependency to be equal to or above `^3.0.0`.
+
+### Make required changes in `webpack.js`.
+
+Add two new dependencies. `fs` will be used to check if the styleguide's version is linked or normally installed and `postcss-import-webpack-resolver` will allow the creation of aliases in postcss.
+
+```js
+const fs = require('fs');
+const createResolver = require('postcss-import-webpack-resolver');
+```
+
+Inside `buildConfig`, before the `return` line, add the following block. This will check that there is a linked version of `discussify-styleguide`
+
+```js
+const existsStyleguideSrc = fs.existsSync(path.join(projectDir, 'node_modules/@discussify/styleguide/src'));
+```
+
+#### In the returned object
+
+In the `resolve` attribute, add the following attribute to the object. Create an alias so that your project will resolve the `import`s to the styleguide in the package's `src/` instead of `dist/`. This will allow your project to update when changes occur to `js` files in the styleguide, without requiring a new styleguide build.
+
+```js
+// In dev, we want to compile the styleguide when linked
+alias: isDev && existsStyleguideSrc ? {
```

Comment:
isDev is specific to the browser extension project, change to `process.env.NODE_ENV = ‘development’ or declare it above.